package.json

```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "addressit": "git://github.com/dianashk/addressit.git#temp",
     "extend": "^3.0.0",
-    "lodash": "^4.13.1",
+    "lodash": "4.14.0",
```

Comment:
doh, this one can stay as a caret version. Being a big well known package they're likely to adhere strictly to semver, but they release [almost every day](https://github.com/lodash/lodash/releases) and we'd be drowned in a sea of Greenkeeper PRs